src/css/style.css

```
@@ -166,3 +165,36 @@
+  to   { opacity: 1; }
+}
+.animate__fadeIn {
+  animation-name: fadeIn;
+}
+
+@keyframes bounceIn {
+  from, 20%, 40%, 60%, 80%, to {
+    animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
+  }
+  0%   { opacity: 0; transform: scale3d(0.3, 0.3, 0.3); }
+  20%  { transform: scale3d(1.1, 1.1, 1.1); }
```

Stylelint (17.3.0)

[error] Unexpected duplicate "20%" (keyframe-block-no-duplicate-selectors)

(keyframe-block-no-duplicate-selectors)